src/platform/web/dom/History.js

```diff
@@ -17,6 +17,12 @@ limitations under the License.
 import {BaseObservableValue} from "../../../observable/ObservableValue";
 
 export class History extends BaseObservableValue {
+    
+    constructor() {
+        super();
+        this._lastSessionHash = null;
```

Comment:
Thanks for the change, we try to use `undefined` going forward though as it's nicer to type in typescript.
```suggestion
        this._lastSessionHash = undefined;
```